paasta_tools/kubernetes_tools.py

```diff
@@ -1440,6 +1442,11 @@ def format_kubernetes_app(self) -> Union[V1Deployment, V1StatefulSet]:
     def get_kubernetes_service_account_name(self) -> Optional[str]:
         return self.config_dict.get("service_account_name", None)
 
+    def is_injection_eanbled(self):
```

Comment:
we probably want to add some types here and give this a more descriptive name - maybe something like
```suggestion
    def is_istio_injection_enabled(self) -> Literal["true", "false"]:
```

or
```suggestion
    def should_inject_istio_sidecar(self) -> Literal["true", "false"]:
```

Reply:
`Literal` seem to be only available in python 3.8 and up. Ref https://docs.python.org/3/library/typing.html#typing.Literal

Reply:
rip :(

`-> str` it is until we move to py3.8

Reply:
closing this PR as this is mixed with another one